pkg/config/user_config.go

```diff
@@ -360,6 +361,7 @@ func GetDefaultConfig() *UserConfig {
 				UnstagedChangesColor:      []string{"red"},
 			},
 			CommitLength:             CommitLengthConfig{Show: true},
+			CommitLengthWarning:      72,
```

Comment:
I'd bundle this inside `CommitLengthConfig`. I'd also call it `WarningThreshold`. I'd also default it to disabled (which we can encode via `0`) because people have different opinions about whether commit length matters.